# This is required so that the RPATH information for libgcc/libgomp is
# not stripped during CMake's install phase: -- mandree

--- ./CMakeLists.txt.orig	2013-06-15 19:16:41.000000000 +0200
+++ ./CMakeLists.txt	2013-06-15 19:17:02.000000000 +0200
@@ -6,6 +6,7 @@
 endif (WIN32)
 
 PROJECT(RawTherapee)
+SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 
 # the default target is 'Debug'
 if (CMAKE_BUILD_TYPE STREQUAL "")
